src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age ? (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    ) : (
+      ''
+    )}
```

Comment:
```suggestion
    {person.age && (
      <p className="Person__age">
        {`I am ${person.age}`}
      </p>
    )}
```
no need to render empty string or something else if person.age doesn't provide